packages/core/upload/admin/src/pages/SettingsPage/tests/index.test.js

```diff
@@ -0,0 +1,64 @@
+/**
+ *
+ * Tests for SettingsPage
+ *
+ */
+
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import { ThemeProvider, lightTheme } from '@strapi/parts';
+import { SettingsPage } from '../index';
+import server from './utils/server';
+
+jest.mock('@strapi/helper-plugin', () => ({
+  ...jest.requireActual('@strapi/helper-plugin'),
+  useNotification: jest.fn(),
+  useOverlayBlocker: () => ({ lockApp: jest.fn(), unlockApp: jest.fn() }),
+  useFocusWhenNavigate: jest.fn(),
+}));
+
+const App = (
+  <ThemeProvider theme={lightTheme}>
+    <IntlProvider locale="en" messages={{ en: {} }} textComponent="span">
+      <SettingsPage />
+    </IntlProvider>
+  </ThemeProvider>
+);
+
+describe('Upload | SettingsPage', () => {
+  beforeAll(() => server.listen());
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  afterEach(() => server.resetHandlers());
+
+  afterAll(() => server.close());
+
+  it('renders and matches the snapshot', () => {
+    const { container } = render(App);
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should display the form correctly with the initial values', async () => {
+    const { getAllByTestId } = render(App);
+
+    render(App);
```

Comment:
You've rendered the application twice here.

Reply:
Can't we use the `getByTestId` you will not need to do the `[0]` anymore.

Reply:
I did not see that I was rendering the application twice so I had to use the `getAllByTestId`... Fixing it right now!